File: Kpd37Gomel/Kpd37Gomel.DataAccess/Models/Tenant.cs
using System;

namespace Kpd37Gomel.DataAccess.Models
{
    public class Tenant
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string LastName { get; set; }
        public bool IsAdmin { get; set; }
        public bool IsOwner { get; set; }
        public bool IsDeleted { get; set; }
        public DateTime? DeletionDateUtc { get; set; }

        public Guid ApartmentId { get; set; }
        public Apartment Apartment { get; set; }
    }
}

File: Kpd37Gomel/Kpd37Gomel/ClientApp/src/app/tenant.ts
import { IApartment } from "./apartment";

export interface ITenant {
  id: string;
  firstName: string;
  middleName: string;
  lastName: string;
  isApartmentOwner: boolean;
}

export class Tenant implements ITenant {
  constructor(
    public id: string,
    public firstName: string,
    public middleName: string,
    public lastName: string,
    public isApartmentOwner: boolean) { }
}
